cabby/geo/walk_test.py

```diff
@@ -17,37 +17,53 @@
 import unittest
 from shapely.geometry.point import Point
 from cabby.geo import walk
+from cabby.geo.map_processing import map_structure
 
 
 class WalkTest(unittest.TestCase):
 
     def setUp(self):
-        
+
         # Load map from disk.
-        self.map = map_structure.Map("Manhattan", 18, "/mnt/hackney/data/cabby/poi/")
+        self.map = map_structure.Map(
+            "Manhattan", 18, "/mnt/hackney/data/cabby/poi/v1/")
 
-    def testSingleOutput(self):
+    def testRouteCalculation(self):
         start_point = Point(-73.982473, 40.748432)
         end_point = Point(-73.98403, 40.74907)
-        list_points = walk.compute_route(start_point, end_point, self.map.poi_graph, self.map.nodes)
+        route = walk.compute_route(
+            start_point, end_point, self.map.nx_graph, self.map.nodes)
 
         # Check that two points have arrived.
-        self.assertEqual(len(list_points), 2)
+        self.assertEqual(len(route), 2)
 
-        # Check that the correct points have arrived.
-        first_point = walk.tuple_from_point(list_points[0])
-        second_point = walk.tuple_from_point(list_points[1])
+        # Check that the correct points 
+        first_point = walk.tuple_from_point(route.iloc[0]['geometry'])
+        second_point = walk.tuple_from_point(route.iloc[1]['geometry'])
         self.assertEqual(first_point, (40.749102, -73.984076))
         self.assertEqual(second_point, (40.748432, -73.982473))
```

Comment:
Should these be assertAlmostEqual (on each value)?

Reply:
I made the change.

Reply:
Right -- you need to do each value separately with assertAlmostEqual. I guess it is fine to leave as is for now.